pkgs/applications/editors/emacs/make-emacs.nix

```diff
@@ -221,52 +230,75 @@ mkDerivation (finalAttrs: {
     jansson
     libxml2
     ncurses
-  ] ++ lib.optionals withGconf [
+  ]
+  ++ lib.optionals withGconf [
     gconf
-  ] ++ lib.optionals withAcl [
+  ]
+  ++ lib.optionals withAcl [
     acl
-  ] ++ lib.optionals withAlsaLib [
+  ]
+  ++ lib.optionals withAlsaLib [
     alsa-lib
-  ] ++ lib.optionals withGpm [
+  ]
+  ++ lib.optionals withGpm [
     gpm
-  ] ++ lib.optionals withDbus [
+  ]
+  ++ lib.optionals withDbus [
     dbus
-  ] ++ lib.optionals withSelinux [
+  ]
+  ++ lib.optionals withSelinux [
     libselinux
-  ] ++ lib.optionals (!stdenv.isDarwin && withGTK3) [
+  ]
+  ++ lib.optionals (!stdenv.isDarwin && withGTK3) [
     gsettings-desktop-schemas
-  ] ++ lib.optionals (stdenv.isLinux && withX) [
+  ]
+  ++ lib.optionals (stdenv.isLinux && withX) [
     libotf
     m17n_lib
-  ] ++ lib.optionals (withX && withGTK2) [
+  ]
+  ++ lib.optionals (withX && withGTK2) [
     gtk2-x11
-  ] ++ lib.optionals (withX && withGTK3) [
+  ]
+  ++ lib.optionals (withX && withGTK3) [
     gtk3-x11
-  ] ++ lib.optionals (withX && withMotif) [
+  ]
+  ++ lib.optionals (withX && withMotif) [
     motif
-  ] ++ lib.optionals withGlibNetworking [
+  ]
+  ++ lib.optionals withGlibNetworking [
     glib-networking
-  ] ++ lib.optionals withNativeCompilation [
+  ]
+  ++ lib.optionals withNativeCompilation [
     libgccjit
-  ] ++ lib.optionals withImageMagick [
+  ]
+  ++ lib.optionals withImageMagick [
     imagemagick
-  ] ++ lib.optionals withPgtk [
+  ]
+  ++ lib.optionals withMailutils [
+    mailutils
+  ]
```

Comment:
```suggestion
```

mailutils is not needed at compile-time.

Instead, it is needed at runtime.  So this should be added:

```nix
# Emacs needs to find movemail at run time, see info (emacs) Movemail
propagatedUserEnvPkgs = lib.optional withMailutils mailutils;
```

Reply:
emm, I do not think this is resolved.

Reply:
It is resolved by your latest commit.